src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,30 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def highlight_command(raw_command):
+    """highlight a command to make it colored.
+
+    :param raw_command: The command that needs to be colored
+    :type raw_command: str
+    :return: The styled command text
+    :type: list
+    """
+
+    styled_command = []
+    argument_begins = False
+
+    for index, arg in enumerate(raw_command.split()):
+        spaced_arg = ' {}'.format(arg) if index > 0 else arg
+        style = Style.PRIMARY
+
+        if arg.startswith('-') and '=' not in arg:
+            style = Style.ACTION
+            argument_begins = True
+        elif not argument_begins and '=' not in arg:
+            style = Style.ACTION
```

Comment:
Can you add comment about the rule used?

Reply:
I made some comments at https://github.com/Azure/azure-cli/pull/16257#discussion_r547698672

The logic of `=` handling is a little bit tricky. It works for `az config set a=b`, but it doesn't work for `az config unset output`. It  is the nature of positional argument that we can NOT decide whether an arg is 
- part of a command, or
- a positional argument 

until semantic analysis is involved. 

Let's keep it simple and treat it as part of the command by now.

Reply:
Another possible solution is to utilize `linter_exclusions.yml` which contains the full list of commands with positional arguments:

https://github.com/Azure/azure-cli/blob/2c52cbcef7f43c85088118e672ded491d674737d/linter_exclusions.yml#L1181-L1185

Reply:
This might be a solution, but also making the function much more complex and harder to maintain. 

I prefer we keep it simple, until we really need to consider the case.